zerver/lib/actions.py

```diff
@@ -1499,7 +1499,7 @@ def fetch_stream_subscriber_emails(stream):
             for added_user in new_users:
                 event = dict(type="subscription", op="peer_add",
                              subscriptions=[stream.name],
-                             user_email=added_user.email)
+                             user_id=added_user.id)
```

Comment:
Something we should think about is whether `user_email` should still be sent as a convenience for API clients (it's a lot more `print` friendly).